src/components/Area/Area.js

```diff
@@ -0,0 +1,14 @@
+import  React from 'react';
+import './Area.css';
+
+const Area = (props) => {
```

Comment:
Used props here as opposed to destructuring considering there were so many props - but I'm sure there's a better way to destructure this!